src/Kunstmaan/MenuBundle/Repository/MenuItemRepository.php

```diff
@@ -2,9 +2,30 @@
 
 namespace Kunstmaan\MenuBundle\Repository;
 
+use Doctrine\ORM\EntityManager;
 use Gedmo\Tree\Entity\Repository\NestedTreeRepository;
 
 class MenuItemRepository extends NestedTreeRepository
 {
+    public function getMenuItemsForLanguage(EntityManager $em, $menuName, $locale)
```

Comment:
What is the EntityManager doing here as an argument? It should be removed as this is a repository, which already has access to the entity manager.